packages/common/config.js

```diff
@@ -235,13 +233,37 @@ const getLibraryLocations = () => {
   return files
 }
 
+/**
+ * Get a list of all elements in a factory
+ * @return array containing names and locations and versions
+ */
+const getElements = (factoryLocation) => {
+  try {
+    // look through the listing of workspaces and return a flattened
+    // array of elements
+    return execFile(lernaPath, ["list", "--json"], { cwd: factoryLocation }, (err, stdOut) => JSON.parse(stdOut))
```

Comment:
Does OCLIF automatically handle async?

It may need to become something like:
```suggestion
    return JSON.parse(execFileSync(lernaPath, ["list", "--json"], { cwd: factoryLocation }))
```

if it needs to be sync.